NEXT_CHANGELOG.md

```diff
@@ -3,12 +3,14 @@
 ## Release v0.261.0
 
 ### Notable Changes
+Some of CLI commands such as "update-alert" and etc. now have a required positional argument "UPDATE_MASK"
```

Comment:
there are not so many commands that added this parameter in this PR - perhaps a good idea to list them all instead of using "and etc." phrase